### PR TITLE
Minor documentation changes

### PR DESCRIPTION
**Pull request by jcpitre**
**Summary:**

Added a little bit of documentation related to https://github.com/MobilityData/gtfs-validator/pull/1628

- [ ] Run the unit tests with `gradle test` to make sure you didn't break anything
- [ ] Add or update any needed [documentation](https://github.com/MobilityData/gtfs-validator/tree/master/docs) to the repo 
- [ ] Format the title like "feat: [new feature short description]". Title must follow the Conventional Commit Specification(https://www.conventionalcommits.org/en/v1.0.0/).
- [ ] Linked all relevant issues
- [ ] Include screenshot(s) showing how this pull request works and fixes the issue(s)

### CODE REVIEW

File: docs/USAGE.md [hunk 1]
```diff
@@ -20,7 +20,8 @@
...
-| `-p`       | `--pretty`                    | Optional               | Pretty JSON validation report. If specified, the JSON validation report will be printed using JSON Pretty print. This does not impact data parsing.                                                                                                      |
+| `-p`       | `--pretty`                    | Optional               | Pretty JSON validation report. If specified, the JSON validation report will be printed using JSON Pretty print. This does not impact data parsing.                                                                                                          |
+| `-d`       | `--date`                      | Optional               | Date to simulate when validating, in ISO_LOCAL_DATE format like '2001-01-30'. By default, the current date is used.                                                                                                                                          |
```
Comment (emmambd) [inline docs/USAGE.md:24]: @jcpitre "Date to simulate when validating" is a bit vague. What about "The date used to validate the feed for time-based rules, e.g [`feed_expiration_30_days`](https://gtfs-validator.mobilitydata.org/rules.html#feed_expiration_date30_days-rule)."

Comment (jcpitre) [reply docs/USAGE.md:24]: Sure. I was just copying the text from @bradyhunsaker's changes, but less vague is always good.

**Review by davidgamez — APPROVED** [review]
LGTM 💪 !

**Comment by bradyhunsaker** [comment]
Sorry that I overlooked updating this file with the original change.

**Comment by jcpitre** [comment]
> Sorry that I overlooked updating this file with the original change.

@bradyhunsaker You did a lot already. I'm happy to take care of small details like that,

**Pull request merged**